### PR TITLE
AAFReader: Added kind support for auxiliary AAF tracks (#28)

### LINKED ISSUE #28: AAF Reader: Add support for non-AV track kinds
## Description

Currently the AAF adapter doesn't assign a track `kind` to adapters that aren't audio or video. This makes it harder to work with / filter them.

### PR DESCRIPTION
**Pull request by timlehr**
Fixes #28 

## Description

Makes sure non audio / video tracks get a `kind` string assigned. I added an `AAF_` prefix, followed by the AAF track kind.

### CODE REVIEW

File: src/otio_aaf_adapter/adapters/advanced_authoring_format.py [hunk 1]
```diff
@@ -868,8 +868,8 @@ def _transcribe(item, parents, edit_rate, indent=0):
             elif media_kind in ("SoundMasterTrack", "Sound"):
                 result.kind = otio.schema.TrackKind.Audio
             else:
-                # Timecode, Edgecode, others?
-                result.kind = ""
+                # Timecode, Edgecode, Data, ...
+                result.kind = f"AAF_{media_kind}"
```
Comment (reinecke) [inline src/otio_aaf_adapter/adapters/advanced_authoring_format.py:872]: For reference, [here are the DataDefs](https://github.com/markreidvfx/pyaaf2/blob/main/src/aaf2/model/datadefs.py#L101C7-L101C7) that drive the media kind that's used from @markreidvfx

**Comment by timlehr** [comment]
@reinecke @markreidvfx added a test for this change.

**Comment by reinecke** [comment]
LGTM - But I'll let @markreidvfx have the final 👍 since he's more deeply familiar with the adapter.

**Comment by timlehr** [comment]
@markreidvfx any opinion on this?

**Comment by markreidvfx** [comment]
sorry for the delay. LGTM. 

Something to be aware of is that `media_kind` could have spaces in it. I've seen it be inconsistent between AAF files, namely with the  `Descriptive Metadata` media_kind for markers.  This is would be more of issue for the AAF writer.

**Review by jminor — APPROVED** [review]
From meeting discussion: everyone seems okay with landing this.

**Pull request merged**